src/main/scala/org/broadinstitute/dsde/workbench/gpalloc/api/GPAllocRoutes.scala

```diff
@@ -34,14 +37,41 @@ class GPAllocRoutes()(implicit val system: ActorSystem, val materializer: Materi
       }
     }
 
+  def gpAllocRoutes: Route =
+    requireUserInfo { userInfo =>
+
```

Comment:
Do we get a meaningful message if `request/releaseGoogleProject` calls fail? We should test for those scenarios.

Reply:
the exception handler below turns them into something more web-y. i haven't thought too much about the behaviour google-side, but that shouldn't bubble up here